public/components/ResultsDataTable.vue

```diff
@@ -1,123 +1,122 @@
 <template>
-  <fixed-header-table ref="fixedHeaderTable">
-    <b-table
-      bordered
-      hover
-      small
-      :items="items"
-      :fields="tableFields"
-      :sort-by="errorCol"
-      :sort-compare="sortFunction"
-      @row-clicked="onRowClick"
-      @sort-changed="onSortChanged"
+  <b-table
+    bordered
+    hover
+    small
+    :items="items"
+    :fields="tableFields"
+    :sort-by="errorCol"
+    :sort-compare="sortFunction"
+    @row-clicked="onRowClick"
+    @sort-changed="onSortChanged"
```

Comment:
Here you kept the sorting

Reply:
I need to see if I should've removed it here too.

Reply:
Ah, here because we pass in a special sort which in turn changes what sort it uses based on sort column, we need to listen to event change to update that sort state to pass that information back in the sort function.